package.json

```diff
@@ -74,7 +76,8 @@
     "@sensenet/default-content-types": "^1.2.2",
     "@sensenet/list-controls-react": "^1.3.8",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "react-responsive": "^7.0.0"
```

Comment:
You are not using this library anywhere. Are you going to? if not then it is not needed.

Reply:
Should I move it to "devDependencies" section?